force_wfmanager/ui/setup/mco/base_mco_options_model_view.py

```diff
@@ -22,11 +23,11 @@ class BaseMCOOptionsModelView(ModelView):
     model = Either(Instance(KPISpecification),
                    Instance(BaseMCOParameter))
 
-    #: Only display name and type options for available variables
-    # FIXME: this isn't an ideal method, since it requires further
-    # work arounds for the name validation. Putting better error
-    # handling into the force_bdss could resolve this.
-    available_variables = List(Tuple(Identifier, CUBAType))
+    # Variable selected by the UI to hook model up to
```

Comment:
Here we introduce the `Variable` class as selectable objects for MCO parameters or KPIs. These objects are generated in the `VariableNamesRegistry` and updated each time an input of output slot is edited.